src/libsync/propagateremotemkdir.cpp

```diff
@@ -158,11 +158,18 @@ void PropagateRemoteMkdir::success()
         connect(watcher, &QFutureWatcherBase::finished, this, [watcher, this] {
             const auto *result = watcher->result().get();
             if (*result) {
```

Comment:
Check on `result != nullptr` ?

Reply:
the connect is only triggered if I return something and I really don't intend to return nulll :D